### ISSUE TITLE
Now Playing - Art not showing?

### ISSUE THREAD

**Issue by El-Motu** [kawaiiDango/pScrobbler#108]
Noticed a new issue start happening recently where Various Artist Albums ‘Now Playing’ won’t return the artwork.

However, once the scrobble is done, then no problem, the album art shows, not sure why this happens but I’m guessing it has to do with album artist on the ‘now playing’ script.

Here’s an example: 
![20BB9174-8E5E-4B23-B4E0-6427E5EFDBBC](https://user-images.githubusercontent.com/74815525/101156025-e3e12000-361f-11eb-86e4-1c73e743afca.png)

Thanks for all the hard work!

**Comment by kawaiiDango** [comment]
This is a lastfm issue. The same data is sent to both now playing and scrobble.
